src/python/CRABClient/Commands/submit.py

```diff
@@ -192,13 +192,17 @@ def validateConfig(self):
                 msg = "Invalid CRAB configuration: Parameter General.requestName should not be longer than %d characters." % (requestNameLenLimit)
                 return False, msg
 
+        splitting = getattr(self.configuration.Data, 'splitting', 'Automatic')
+        autoSplitt = True if splitting == 'Automatic' else False
+        autoSplittUnitsMin = parametersMapping['on-server']['minAutomaticRuntimeMins']['default']
+        autoSplittUnitsMax = 2700 # 45 hours
```

Comment:
This parameter is not defined on the server so I did not add it to the `parametersMapping` dictionary, as I did for the one in the line above.
@belforte, do you like it defined here?